CMakeLists.txt

```diff
@@ -5,10 +5,13 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 
 include(FetchContent)
-FetchContent_Declare(SFML
+FetchContent_Declare(sfml
```

Comment:
Any reason to change `SFML` to `sfml`?

Reply:
Lower case is required for any of the variables (e.g. using `sfml_POPULATED` below). It's not technically required for the actual content name declaration to be lowercase, but it's generally good practice and is more consistent for the name to match - It's also what all the examples in the documentation do